google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/HttpApi.scala

```diff
@@ -102,7 +102,7 @@ private trait HttpApi {
   ): Future[immutable.Seq[String]] = {
     import materializer.executionContext
 
-    val url: Uri = s"$PubSubGoogleApisHost/v1/projects/$project/topics/$topic:publish?key=$apiKey"
+    val url: Uri = s"$PubSubGoogleApisHost/v1/projects/$project/topics/$topic:publish"
```

Comment:
Hmm too bad we didn't have a test that touched this part of the flow - can you think of a reasonable one to add?